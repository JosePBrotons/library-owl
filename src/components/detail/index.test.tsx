import React from 'react';
import renderer from 'react-test-renderer';
import Detail from '.';

test('renders login component correctly', () => {
    const detailCmp = renderer.create(<Detail />).toJSON();
    expect(detailCmp).toMatchSnapshot();
});
