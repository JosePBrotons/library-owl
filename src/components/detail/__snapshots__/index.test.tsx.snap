// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders book detail component correctly 1`] = `
<RCTScrollView
  style={
    Object {
      "backgroundColor": "rgba(235, 246, 251, 1)",
      "flex": 1,
      "flexDirection": "column",
      "paddingHorizontal": 20,
    }
  }
>
  <View>
    <View
      style={
        Object {
          "backgroundColor": "rgba(255, 255, 255, 1)",
          "borderRadius": 5,
          "elevation": 3,
          "flex": 1,
          "marginVertical": 10,
          "padding": 15,
          "shadowColor": "rgba(69, 69, 69, 0.5)",
          "shadowOffset": Object {
            "height": 2,
            "width": 2,
          },
          "shadowOpacity": 0.5,
          "shadowRadius": 3,
          "width": "100%",
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "justifyContent": "center",
            "paddingHorizontal": 20,
          }
        }
      >
        <Image
          source={
            Object {
              "uri": "",
            }
          }
          style={
            Object {
              "height": 100,
              "marginHorizontal": 20,
              "width": 70,
            }
          }
        />
        <View
          style={
            Object {
              "alignItems": "flex-start",
            }
          }
        >
          <Text
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Object {
                "color": "rgba(69, 69, 69, 1)",
                "fontFamily": "Nunito-Bold",
                "fontSize": 16,
                "lineHeight": 22,
                "width": "100%",
              }
            }
          >
            
          </Text>
          <Text
            style={
              Object {
                "color": "rgba(76, 175, 80, 1)",
                "fontFamily": "Nunito-Regular",
                "fontSize": 16,
                "lineHeight": 22,
              }
            }
          >
            [missing "en.bookDetail.available" translation]
          </Text>
          <Text
            ellipsizeMode="tail"
            numberOfLines={2}
            style={
              Object {
                "color": "rgba(106, 106, 106, 1)",
                "fontFamily": "Nunito-Regular",
                "fontSize": 14,
                "lineHeight": 22,
              }
            }
          >
            
          </Text>
          <Text
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Object {
                "color": "rgba(106, 106, 106, 1)",
                "fontFamily": "Nunito-Regular",
                "fontSize": 14,
                "lineHeight": 22,
              }
            }
          >
            
          </Text>
          <Text
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Object {
                "color": "rgba(106, 106, 106, 1)",
                "fontFamily": "Nunito-Regular",
                "fontSize": 14,
                "lineHeight": 22,
              }
            }
          >
            
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "column",
            "marginVertical": 15,
          }
        }
      >
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "rgba(255, 255, 255, 1)",
              "borderColor": "rgba(57, 172, 233, 1)",
              "borderRadius": 20,
              "borderWidth": 1,
              "height": 40,
              "justifyContent": "center",
              "marginBottom": 10,
              "marginTop": 5,
              "opacity": 1,
              "paddingHorizontal": 10,
              "width": "70%",
            }
          }
        >
          <Text
            numberOfLines={1}
            style={
              Object {
                "color": "rgba(57, 172, 233, 1)",
                "fontFamily": "Nunito-Bold",
                "fontSize": 16,
                "lineHeight": 22,
              }
            }
          >
            [missing "en.bookDetail.wishlist" translation]
          </Text>
        </View>
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "rgba(57, 172, 233, 1)",
              "borderRadius": 20,
              "height": 40,
              "justifyContent": "center",
              "marginBottom": 10,
              "marginTop": 5,
              "opacity": 1,
              "paddingHorizontal": 10,
              "width": "70%",
            }
          }
        >
          <Text
            numberOfLines={1}
            style={
              Object {
                "color": "rgba(255, 255, 255, 1)",
                "fontFamily": "Nunito-Bold",
                "fontSize": 16,
                "lineHeight": 22,
              }
            }
          >
            [missing "en.bookDetail.rent" translation]
          </Text>
        </View>
      </View>
    </View>
  </View>
</RCTScrollView>
`;
